File: src/main/java/com/example/ssauc/admin/repository/AdminRepository.java
package com.example.ssauc.admin.repository;

import com.example.ssauc.admin.entity.Admin;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.Optional;

public interface AdminRepository extends JpaRepository<Admin, Long> {
    Optional<Admin> findByEmail(String email);

    Optional<Admin> findByEmailAndPassword(String email, String password);
}

File: src/main/java/com/example/ssauc/admin/repository/AdminRepository.kt
package com.example.ssauc.admin.repository

import com.example.ssauc.admin.entity.Admin
import org.springframework.data.jpa.repository.JpaRepository
import java.util.*

interface AdminRepository : JpaRepository<Admin?, Long?> {
    fun findByEmail(email: String?): Optional<Admin?>?

    fun findByEmailAndPassword(email: String?, password: String?): Optional<Admin?>?
}